repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -91,13 +90,13 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
 
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, chain: FilterChain): Unit = {
     /**
-     * STATIC REFERENCE TO CONFIG
-     */
+      * STATIC REFERENCE TO CONFIG
```

Comment:
you went from the way it more commonly is to this, why? Is the standard different in scala, because that would be stupid and be them just trying to be differnt